src/data/quickstarts.json

```diff
@@ -3391,6 +3394,7 @@
         "url": "https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.12.0/packs/gatsby-build/dashboards/gatsby-build.json"
       }
     ],
+    "keywords": ["featured"],
```

Comment:
What occurs when we actually merge in the quick start data and the actual keywords are set, will these be removed? I'm asking as these aren't actual featured QSs, so just want to make sure this is just for testing reasons.

Reply:
Ye when we manually update with proper keywords, any test keywords i've entered will be removed so no stress